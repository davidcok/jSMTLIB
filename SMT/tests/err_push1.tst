; push command before a set-logic
(push 0)
