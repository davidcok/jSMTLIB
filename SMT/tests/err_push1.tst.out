(push 0)
^^^^^^^^
(error "The logic must be set before a push command is issued")
